2016-EDS/GeocodeAndRoute/Readme.md

```diff
@@ -0,0 +1,38 @@
+Geocode and Route
+=======================
+This app uses a local data to demonstrate an offline workflow. A mobile map package is used to display a map with operational data, and perform geocoding and routing. A tile cache is also used to display a basemap.
+
+This was used in the 'ArcGIS Runtime SDKs: Working with apps online and offline' technical session at the Esri Developer Summit Europe, December 2016.
+
+![GeocodeAndRoute Screenshot](Screenshot_GeocodeAndRoute.png)
+
+## APIs used
+- `MobileMapPackage`
+- `ArcGISTiledLayer`
+- `LocatorTask`
+- `RouteTask`
+
+## Prerequisites
+- The data used in this demo is not available for distribution. The code is therefore provided for information only, and can be adapted to your own datasets. The app requires a mobile map package (.mmpk) containing at least one map, with a corresponding locator and network dataset combined with operational data in the map. A tile cache (.tpk) file is required to be used as the basemap layer for the map. The code assumes specific locator names in order to assign the geocoded location as either an address or a hydrant location.
```

Comment:
You should explicitly explain where to make edits in code to include your own mmpk and tpk files.  You should also note the viewpoint may need to be adjusted to your dataset.